pkg/ui/src/redux/cachedDataReducer.ts

```diff
@@ -52,11 +52,13 @@ export class CachedDataReducer<TRequest, TResponseMessage> {
    * actionNamespace - A unique namespace for the redux actions.
    * invalidationPeriod (optional) - The duration after
    *   data is received after which it will be invalidated.
+   * requestTimeout (optional)
    */
   constructor(
     protected apiEndpoint: APIRequestFn<TRequest, TResponseMessage>,
     public actionNamespace: string,
     protected invalidationPeriod?: moment.Duration,
+    protected requestTimeout?: moment.Duration,
```

Comment:
should this default to the `invalidationPeriod` to keep the old behavior maybe?  or was that design flawed anyway, and no default is better?